test/handlers/test-pre-login-handler.js

```diff
@@ -145,33 +148,29 @@ describe('Pre-Login Handler', function () {
         });
 
         describe('when providing your own response', function () {
-          var expressApp, customResponse;
-
-          before(function (done) {
-            customResponse = { test: '6ba109b3-2cdf-421c-8cc0-e22bc7550d0b' };
-            preLoginHandlerInterceptResponseTestFixture(application, customResponse, function (fixture) {
-              expressApp = fixture.expressApp;
-              done();
-            });
-          });
-
           it('should return that response', function (done) {
-            request(expressApp)
-              .post('/login')
-              .set('Accept', 'application/json')
-              .type('json')
-              .send(usernamePasswordBody)
-              .expect(200)
-              .end(function (err, res) {
-                if (err) {
-                  return done(err);
-                }
-
-                assert(typeof res.body === 'object');
-                assert(res.body.test === customResponse.test);
-
-                done();
-              });
+            preLoginHandlerInterceptResponseTestFixture(application, function (fixture) {
+              request(fixture.expressApp)
+                .post('/login')
+                .set('Accept', 'application/json')
+                .type('json')
+                .send(usernamePasswordBody)
+                .expect(200)
+                .end(function (err) {
+                  if (err) {
+                    return done(err);
+                  }
+
+                  var preLoginHandlerFormData = fixture.preLoginHandlerFormData;
```

Comment:
The intention of this test case is to assert that the developer can end the response inside of their preLoginHandler, and that their response will be the final one.

These modifications don't assert that code path anymore, because the `preLoginHandlerInterceptResponseTestFixture` has been modified to always call `next()`, so we aren't testing a custom response by the developer anymore.  This test should be reverted back to it's original intent.